templates/nextjs/package.json

```diff
@@ -31,6 +31,7 @@
     "eslint-config-next": "14.2.4",
     "postcss": "^8",
     "tailwindcss": "^3.4.4",
-    "typescript": "^5"
+    "typescript": "^5",
+    "vitest": "^1.6.0"
```

Comment:
Do we also need a vitest config file?

Reply:
Not if we're happy with the defaults, which given the simplicity of this setup, I think we don't need one.